rlclientlib/live_model_impl.h

```diff
@@ -96,11 +96,10 @@ namespace reinforcement_learning
 
     std::unique_ptr<model_management::i_data_transport> _transport{nullptr};
     std::unique_ptr<model_management::i_model> _model{nullptr};
-    std::unique_ptr<logger::cb_logger_facade> _ranking_logger{nullptr};
-    std::unique_ptr<logger::ca_logger_facade> _continuous_action_logger{nullptr};
+
+    std::unique_ptr<logger::interaction_logger_facade> _ranking_logger{nullptr};
```

Comment:
I am wondering if this should be named more broadly `_interaction_logger` since its type is of `interaction_logger_facade` and ranking doesn't really make sense in the continuous actions context

Reply:
agree, let me fix it